requirements.txt

```
@@ -1,4 +1,4 @@
 ansible
 docker
 jinja2
-molecule-docker==2.0.0
+git+https://github.com/ansible-community/molecule-docker.git@main#egg=molecule-docker
```

could we pin to a commit please? this is the latest commit on master currently:
```suggestion
git+https://github.com/ansible-community/molecule-docker.git@5b26cc6cedb59de813229fd8a5bed7eb194db230#egg=molecule-docker
```

---

Pleas do not make requests of me without providing the reasoning behind the request. And no I am not going to track a specific commit when nothing is updating it automatically to a newer version. If someone set up Dependabot for this file then I would be fine with a deterministic build.

---

The reason is for deterministic CI builds, molecule has had breakages in more than one version why track their main branch?